metals/src/main/scala/scala/meta/internal/tvp/IndexedSymbols.scala

```diff
@@ -82,6 +64,40 @@ class IndexedSymbols(isStatisticsEnabled: Boolean)(implicit rc: ReportContext)
     result
   }
 
+  private def workspaceSymbolsFromPath(
+      in: AbsolutePath
+  ): Array[TreeViewSymbolInformation] =
+    trees
+      .get(in)
+      .map { tree =>
+        (tree, tree.origin) match {
+          case (
+                src: Source,
+                Origin.Parsed(input: Input.VirtualFile, dialect, _),
+              ) =>
+            val mtags = new ScalaMtags(input, dialect, Some(src))
```

Comment:
I see we only show for scala build targets but maybe we could use java mtags to show for java as well?